### PR TITLE
[FEAT] Add google autocomplete

### PR DESCRIPTION
**Pull request by MikeVedsted**
Add google autocomplete box for address selection. 

#92

### CODE REVIEW

File: client/src/components/GoogleAutoComplete/index.tsx [hunk 1]
```diff
@@ -1,24 +1,57 @@
...
+  const resetSelection = async () => {
+    address.street = ''
+    address.number = ''
+    address.city = ''
+    address.postal_code = ''
+    address.country = ''
+    address.lat = ''
+    address.lng = ''
+    setFormattedAddress('')
+  }
```
Comment (Rostislav013) [inline client/src/components/GoogleAutoComplete/index.tsx:30]: Do u need async here?

File: client/src/components/GoogleAutoComplete/index.tsx [hunk 2]
```diff
@@ -1,24 +1,57 @@
...
+        address.number = value.long_name
+      if (value.types.includes('route')) address.street = value.long_name
+      if (value.types.includes('locality')) address.city = value.long_name
+      if (value.types.includes('country')) address.country = value.long_name
+      if (value.types.includes('postal_code'))
+        address.postal_code = value.long_name
     })
+    setFormattedAddress(selection.formatted_address)
+    return address 
+  }
```
Comment (Rostislav013) [inline client/src/components/GoogleAutoComplete/index.tsx:44]: Is async needed here?

**Review by Rostislav013 — APPROVED** [review]

File: client/src/components/GoogleAutoComplete/index.tsx (hunk 1 above)
Comment (MikeVedsted) [reply client/src/components/GoogleAutoComplete/index.tsx:30]: I guess not.. I could probably also refactor lines 22-28.

File: client/src/components/GoogleAutoComplete/index.tsx (hunk 2 above)
Comment (MikeVedsted) [reply client/src/components/GoogleAutoComplete/index.tsx:44]: not really 😄

Comment (MikeVedsted) [reply client/src/components/GoogleAutoComplete/index.tsx:44]: On second thought, if I want to await it in handleSelect, I actually do.. Same for resetSelection, which maybe doesn't need await (but it is nicer to do handleSelect step-by-step..)

**Review by nannadao — APPROVED** [review]

**Review by MarinaFroes — APPROVED** [review]

**Pull request merged**